topics/bzflag/index.md

```diff
@@ -0,0 +1,11 @@
+---
+display_name: BZFlag
+logo: bzflag.png
+short_description: 3D multiplayer tank game created in 1992.
+topic: bzflag
+url: https://bzflag.org
+wikipedia_url: https://en.wikipedia.org/wiki/BZFlag
+---
+Everyone knows that having a tank that can _just_ fly is pretty boring. What if we told you tanks could be equipped with super powers? You'd think we're crazy, right? We were so busy thinking on how we could, that we didn't stop to think whether or not we should. Super powers for all your tanks!
+
+BZFlag is a 3D multiplayer tank game first made in 1992. It runs on all three major platforms and was chosen for the Google Summer of Code three times, in 2007, 2008, and 2009.
```

Comment:
While I think this is fun, I think we should avoid editorializing here.
```suggestion
BZFlag is a 3D multiplayer tank game first made in 1992. It runs on all three major platforms and was chosen for the Google Summer of Code three times, in 2007, 2008, and 2009.
```

Reply:
I was trying to copy the website's "description", but it makes sense not to.